### DISCUSSION TITLE
Render window in tabs with different layout

### DISCUSSION THREAD

**Discussion by AnthonyTSV** [Kitware/trame#416] [General] [answered]
Hi! I'm figuring out if having the same render window instance across different tabs with different layouts is possible. The idea is the following: 
first tab - large render window 
![image](https://github.com/Kitware/trame/assets/93186473/2e6bf896-ab73-4c97-ac63-142a72cac0cc)
second tab - render window with a chart next to it
![image](https://github.com/Kitware/trame/assets/93186473/29458d4b-6da2-40e3-aeb2-bd2ec28a77b6)
Any help is appreciated! Have a beautiful day!

**Comment by jourdain** [comment, marked as answer]
Just use the tab model to trigger some v-show for the chart.
